TestSupport/tables/6/n6.mALL.3.out.txt

```diff
@@ -0,0 +1,20 @@
+| Input filename | # Edges | # Graphs | # OK | # NONEMBEDDABLE |Error flag| Duration | # Invalid OK | EDA Proc Time | EDA Total Time |
+|================|=========|==========|======|=================|==========|==========|==============|===============|================|
+|n6.m0.g6        |0        |1         |1     |0                |SUCCESS   |0.0       |0             |0.016          |0.021           |
+|n6.m1.g6        |1        |1         |1     |0                |SUCCESS   |0.0       |0             |0.000          |0.021           |
+|n6.m2.g6        |2        |2         |2     |0                |SUCCESS   |0.0       |0             |0.000          |0.043           |
+|n6.m3.g6        |3        |5         |5     |0                |SUCCESS   |0.0       |0             |0.031          |0.111           |
+|n6.m4.g6        |4        |9         |9     |0                |SUCCESS   |0.0       |0             |0.000          |0.193           |
+|n6.m5.g6        |5        |15        |15    |0                |SUCCESS   |0.0       |0             |0.094          |0.328           |
+|n6.m6.g6        |6        |21        |21    |0                |SUCCESS   |0.0       |0             |0.109          |0.433           |
+|n6.m7.g6        |7        |24        |24    |0                |SUCCESS   |0.016     |0             |0.094          |0.506           |
+|n6.m8.g6        |8        |24        |24    |0                |SUCCESS   |0.0       |0             |0.172          |0.516           |
+|n6.m9.g6        |9        |21        |20    |1                |SUCCESS   |0.0       |0             |0.125          |0.434           |
+|n6.m10.g6       |10       |15        |14    |1                |SUCCESS   |0.016     |0             |0.109          |0.381           |
+|n6.m11.g6       |11       |9         |7     |2                |SUCCESS   |0.0       |0             |0.062          |0.354           |
```

Comment:
I believe Asano had a theorem stating that a graph with no `K_{3,3}` had to have 3n-5 or fewer edges. So, starting with 3*6-4=14 edges, we need to solve bugs until the 7 OK results here becomes 0 (unless there is a problem with Asano's result, which seems doubtful).